src/map.js

```diff
@@ -0,0 +1,25 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import  Container  from 'react-bootstrap/Container';
+import { Component } from 'react';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import beastBios from './data.json';
+
+
+class Map extends Component {
```

Comment:
Same here, rename this component (typically components are named of view they are showing or something similar)

I believe that was a requirement of lab2, but I will double check.